d2l-floating-buttons.html

```diff
@@ -82,7 +82,12 @@
 			is: 'd2l-floating-buttons',
 
 			properties: {
-				_heightAtMost500px: Boolean
+				minHeight: {
+					reflectToAttribute: true,
+					type: String,
+					value: '500px'
+				},
+				_viewportIsMinHeight: Boolean
```

Comment:
Perhaps `_hasMinHeight` would be better, or perhaps `_isMinHeightCriteriaMet`?  This is the true/false whether the media-query matched... usually I try to express a true/false as either starting with `is`, or `has`.  It's private, so not a big deal, although whatever is going to make it easy for the next person to understand.